internal/graph/check.go

```diff
@@ -38,6 +39,7 @@ type ResolveCheckRequest struct {
 	Context              *structpb.Struct
 	ResolutionMetadata   *ResolutionMetadata
 	VisitedPaths         map[string]struct{}
+	DispatchCounter      *atomic.Uint32
```

Comment:
We already keep track of this number within the ResolutionMetadata https://github.com/openfga/openfga/blob/d7f77478e8024f3f299c66d7279f9a0ccd84ba28/internal/graph/graph.go#L57-L61

Can we remove line 42? If not, can you leave a godoc explaining how these counters differ?